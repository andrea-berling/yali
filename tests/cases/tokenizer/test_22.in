if for and or else fun print ELSE THIS NIL this AND false true class nil FUN OR return FOR RETURN super SUPER WHILE while CLASS FALSE TRUE VAR IF var PRINT
