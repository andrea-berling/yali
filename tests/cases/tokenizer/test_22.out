IF if null
FOR for null
AND and null
OR or null
ELSE else null
FUN fun null
PRINT print null
IDENTIFIER ELSE null
IDENTIFIER THIS null
IDENTIFIER NIL null
THIS this null
IDENTIFIER AND null
FALSE false null
TRUE true null
CLASS class null
NIL nil null
IDENTIFIER FUN null
IDENTIFIER OR null
RETURN return null
IDENTIFIER FOR null
IDENTIFIER RETURN null
SUPER super null
IDENTIFIER SUPER null
IDENTIFIER WHILE null
WHILE while null
IDENTIFIER CLASS null
IDENTIFIER FALSE null
IDENTIFIER TRUE null
IDENTIFIER VAR null
IDENTIFIER IF null
VAR var null
IDENTIFIER PRINT null
EOF  null
